general/solomiia_tymoshchuk/Task3.java

```diff
@@ -0,0 +1,25 @@
+package general.solomiia_tymoshchuk;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a double number and outputs only its
+ * fractional part (all digits after the decimal point).
+ **/
+
+public class Task3 {
+
+    public float getFractionalPart() {
+        System.out.println("Enter the digit");
+        Scanner sc = new Scanner(System.in);
+        float number = sc.nextFloat();
+        float fractionalPart = number % 1;
+        System.out.println(fractionalPart);
+        return fractionalPart;
```

Comment:
similar to previous tasks: if the method returned value won't be used, it's better to not define it